ichnaea/data/public.py

```diff
@@ -15,6 +15,15 @@
 
 def write_stations_to_csv(session, path, today,
                           start_time=None, end_time=None):
+    linesep = '\r\n'
+
+    header_row = [
```

Comment:
This section should not have been moved, it makes it harder to understand what changed.